framework/src/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -101,6 +101,26 @@
             .compile("^\"([^\"\\\\]|\\\\.)*\"$");
     /** Matches the null literal */
     protected static final Pattern nullPattern = Pattern.compile("^null$");
+    /** Matches an expression contained in matching start and end parentheses */
+    protected static final Pattern parenthesesPattern = Pattern.compile("^\\((.*)\\)$");
```

Comment:
There are two other places in here that define an `itself` patter. Should they all be put into this class?